richtextfx-demos/src/main/java/org/fxmisc/richtext/demo/hyperlink/HyperlinkOps.java

```diff
@@ -0,0 +1,41 @@
+package org.fxmisc.richtext.demo.hyperlink;
+
+import org.fxmisc.richtext.model.SegmentOpsBase;
+
+public class HyperlinkOps<S> extends SegmentOpsBase<Hyperlink<S>, S> {
```

Comment:
Where does SegmentOpsBase come from? I don't see it in here, and my compiler isn't finding it.